roles/common/tasks/main.yml

```diff
@@ -4,8 +4,8 @@
   with_items: "{{ trusted_networks }}"
 - name: Set hostname to fqdn
   hostname: name='{{ inventory_hostname }}'
-#- name: Dont bind hostname to loopback
-#  replace: dest=/etc/hosts regexp='^.*{{ inventory_hostname }}.*$' replace='' backup=yes
+- name: Dont bind hostname to loopback
+  replace: dest=/etc/hosts regexp='^(127.0.0.1).*{{ inventory_hostname }}.*$' replace='' backup=yes
```

Comment:
It should now be covered by:
https://github.com/nlesc-sherlock/emma/commit/750aa7018d86acb6fbcf27b2b1db37e7501cb766